docs/develop/manifest-element-ordering.md

```diff
@@ -0,0 +1,50 @@
+
+> [!NOTE]
+> The [Office Add-in Validator](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-manifest#validate-your-manifest-with-the-office-add-in-validator) uses the same error message when an element is out-of-order as it does when an element is under the wrong parent. The error will say that the child element is is not a valid child of the parent element. If you get such an error but the reference documentation for the child element indicates that it *is* valid for the parent, then the problem is likely that the child has been placed in the wrong order.
+
+The required ordering is specified in the XSD files in the [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/docs/overview/schemas) folder. The XSD files are categorized into subfolders for taskpane, content, and mail add-ins.
+
+To find the correct order for the child elements of a given parent element, take the following steps. (This is a simplified process, but XSD files are quite complex. If these steps don't work, there are many resources on the Internet about reading XSD files.)
+
+1. Open the subfolder under [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/docs/overview/schemas) for the type of add-in that you are creating. 
+2. Open the XSD file where the parent element is defined as a complex type. (If you don't know which file has the definition, you may have to do step 3 on multiple files till you find it.)
+3. Search for `<xs:complexType name="PARENT_ELEMENT">`, where PARENT_ELEMENT is the name of the parent element.
+4. Inside the definition for the PARENT_ELEMENT, there is (usually) an element called `<xs:sequence>`. The following is the definition for `<SuperTip>` from [TaskPaneAppVersionOverridesV1_0.xsd](https://raw.githubusercontent.com/OfficeDev/office-js-docs-pr/master/docs/overview/schemas/taskpane/TaskPaneAppVersionOverridesV1_0.xsd).
+
+```xml
+  <xs:complexType name="Supertip">
+    <xs:annotation>
+      <xs:documentation>
+        Specifies the super tip for this control.
+      </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="Title" type="bt:ShortResourceReference" minOccurs="1" maxOccurs="1" />
+      <xs:element name="Description" type="bt:LongResourceReference" minOccurs="1" maxOccurs="1" />
+    </xs:sequence>
+  </xs:complexType>
+```
+
+The `<xs:sequence>` lists the possible child elements, *in the order in which they must appear*. This does *not* necessarily mean that all of them are mandatory. If the `minOccurs` value for a child element is **0**, then the child element is optional. *But if it is present, it must be in the order specified by the `<xs:sequence>` element*.
```

Comment:
"This does *not* necessarily mean that all of them are mandatory." -> "This does *not* mean all of them are mandatory."

You could also get away with removing that second sentence and changing the third to "[…] then the child element is optional and can be omitted."

Reply:
As an entirely different alternative, you could define the `minOccurs` and `maxOccurs` attributes, mentioning what a min-0 implies.